src/Arcus.Scripting.Tests.Integration/Files/bicep-template-object-value (linux).json

```diff
@@ -0,0 +1,3 @@
+﻿{
```

Comment:
Good stuff on using the correct line characters.
Wondering if we can re-use the ones from the ARM template and renaming the object-value files to something more general like `template-object-value (linux).json` for example?
Or do you imagine this will cause some confusion? Fine by me to have separate ones.

Reply:
Probably best to separate them, just to prevent confusion.